File: src/ast/StdQualifiedName.java
package ast;

import java.io.StringWriter;

public class StdQualifiedName extends ParentNode {
    public StdQualifiedName(BaseNode child) {
        super(NodeType.StdQualifiedName, child);
    }

    @Override
    public void PrintLeft(StringWriter writer) {
        writer.write("std::");
        child.Print(writer);
    }
}

File: src/de/fearlesstobi/demangler/ast/StdQualifiedName.kt
package de.fearlesstobi.demangler.ast

import java.io.StringWriter

class StdQualifiedName(child: BaseNode?) : ParentNode(NodeType.StdQualifiedName, child) {
    public override fun printLeft(writer: StringWriter) {
        writer.write("std::")
        child!!.print(writer)
    }
}